components/Pagination.js

```diff
@@ -0,0 +1,34 @@
+import Link from '@/components/Link'
+
+export default function Pagination({ totalPages, currentPage }) {
+  const prevPage = parseInt(currentPage) - 1 > 0
+  const nextPage = parseInt(currentPage) + 1 <= parseInt(totalPages)
+
+  return (
+    <nav className="flex justify-between">
+      {!prevPage && (
+        <button rel="previous" className="cursor-auto">
+          <strike>Previous</strike>
+        </button>
+      )}
+      {prevPage && (
+        <Link href={currentPage - 1 === 1 ? `/blog/` : `/blog/page/${currentPage - 1}`}>
+          <button rel="previous">Previous</button>
```

Comment:
How about using the button disabled state instead of strikethrough? Then you can just do 
```
<button rel="previous" disabled={!prevPage}> Previous </button>
```

Reply:
I replaced `strikethrough` with `disabled={!prevPage}` and also added a class with the value of `cursor-auto disabled:opacity-50`